src/main/java/com/hivemq/bootstrap/netty/ChannelDependencies.java

```diff
@@ -415,4 +416,9 @@ public ConnectInboundInterceptorHandler getConnectInboundInterceptorHandler() {
     public ConnackOutboundInterceptorHandler getConnackOutboundInterceptorHandler() {
         return connackOutboundInterceptorHandler;
     }
+
+    @NotNull
+    public PingRequestResponseInterceptorHandler getPingRequestResponseInterceptorHandler() {
```

Comment:
Does this need to be provided? I think just one single instance is enough and the Handler can be @Sharable .